velox/type/DecimalUtil.cpp

```diff
@@ -49,11 +49,180 @@ std::string formatDecimal(uint8_t scale, int128_t unscaledValue) {
   return fmt::format(
       "{}{}{}", isNegative ? "-" : "", integralPart, fractionString);
 }
+
+/**
```

Comment:
Coding guidelines require /// for multi line comments
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md